PWGGA/PHOSTasks/PHOS_gamma/AliAnalysisTaskGammaPHOSPbPbRun2.cxx

```diff
@@ -591,6 +591,9 @@ void AliAnalysisTaskGammaPHOSPbPbRun2::UserExec(Option_t *)
     averageRP/=2.;
   Int_t irp=Int_t(10.*averageRP*fHarmonics/TMath::TwoPi());
   if(irp>9)irp=9 ;
+
+  irp = 0; //don't need it now
```

Comment:
Using reaction plane binnig considerably improves flatness of Real/Mixed ratio. Are you sure, you do not need it?